extensions/wikia/MarkWikiAsClosed/MarkWikiAsClosed.setup.php

```diff
@@ -0,0 +1,12 @@
+<?php
+$wgExtensionCredits['markwikiclosed'][] = [
```

Comment:
I don't think `markwikiclosed` is a valid [extension type](https://www.mediawiki.org/wiki/Manual:Extension.json/Schema#type).

Reply:
that is correct. Changing to 'api' now